web/packages/api/src/environment.ts

```diff
@@ -183,7 +183,7 @@ export const SNOWBRIDGE_ENV: { [id: string]: SnowbridgeEnvironment } = {
                     {
                         id: "WETH",
                         address: "0xfff9976782d46cc05630d1f6ebab18b2324d6b14",
-                        minimumTransferAmount: 1n,
+                        minimumTransferAmount: 15_000_000_000_000n,
```

Comment:
This line also: 

https://github.com/Snowfork/snowbridge/pull/1234/files#diff-0c721ef12fe6fb53cd42b43781e67cd2000aa1927affa80e849954d3cdca186cL217